policyengine_us/variables/gov/states/ks/tax/income/ks_taxable_income.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class ks_taxable_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "KS taxable income"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.ksrevenue.gov/pdf/ip21.pdf"
+        "https://www.ksrevenue.gov/pdf/ip22.pdf"
+    )
+    defined_for = StateCode.KS
+
+    def formula(tax_unit, period, parameters):
+        std_ded = tax_unit("ks_standard_deduction", period)
+        itm_ded = tax_unit("ks_itemized_deductions", period)
+        deductions = where(itm_ded > std_ded, itm_ded, std_ded)
```

Comment:
Could you confirm then that Kansas itemization is not tied to federal itemization, and leave a comment indicating it if so?

Reply:
Add comment.